### PR TITLE
Update to v1.3 and pull from pypi.io

### PR DESCRIPTION
**Pull request by synapticarbors**

### CODE REVIEW

**Comment by jakirkham** [comment]
Needs a re-render. That should fix the current failures.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+{% set hash_val = "d160c2ce4f8b1831dabfe766bd844cf9012f766539cf84139c2faac5201882ce" %}
 
 package:
-  name: tzlocal
+  name: {{ name }}-{{ version }}.tar.gz
   version: {{ version }}
 
 source:
   fn: tzlocal-{{ version }}.tar.gz
-  url: https://github.com/regebro/tzlocal/archive/{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipe/meta.yaml]: I really prefer the canonical source at GitHub. But that is OK if really want PyPI for some reason.

Comment (synapticarbors) [inline recipe/meta.yaml]: I can switch it back.

Comment (ocefpaf) [inline recipe/meta.yaml]: Thanks. I find copying-n-pasting the URL much easier too without all that `jinja`. At the end of the day we are checking checksums, license files, creating patches, and all that we need to download the tarball locally. While that `jinja` only helps to copy-and-paste to crate another recipe. (I do like the `jinja` for version though, that is the only one that helps to maintain the feedstock.)

**Comment by synapticarbors** [comment]
@jakirkham Typically before I update a feedstock, I re-render the recipe but this time it failed. I updated `conda smithy`:

```
$ conda list conda-build
# packages in environment at _:
#
conda-build               2.1.1                    py27_0
conda-build-all           1.0.0                    py27_0    conda-forge
```

and:

```
$ conda list conda-smithy
# packages in environment at /Users/lev/anaconda:
#
conda-smithy              2.0.0                    py27_0    conda-forge
```

and then running `conda smithy rerender`, I got:

```
$ conda smithy rerender
No circle token.  Create a token at https://circleci.com/account/api and
put it in ~/.conda-smithy/circle.token
No appveyor token. Create a token at https://ci.appveyor.com/api-token and
Put one in ~/.conda-smithy/appveyor.token
No anaconda token. Create a token via
  anaconda auth --create --name conda-smithy --scopes "repos conda api"
and put it in ~/.conda-smithy/anaconda.token
Fetching package metadata ...............
Traceback (most recent call last):
  File "/Users/x/anaconda/bin/conda-smithy", line 6, in <module>
    sys.exit(conda_smithy.cli.main())
  File "/Users/x/anaconda/lib/python2.7/site-packages/conda_smithy/cli.py", line 289, in main
    args.subcommand_func(args)
  File "/Users/x/anaconda/lib/python2.7/site-packages/conda_smithy/cli.py", line 200, in __call__
    configure_feedstock.main(args.feedstock_directory)
  File "/Users/x/anaconda/lib/python2.7/site-packages/conda_smithy/configure_feedstock.py", line 644, in main
    render_run_docker_build(env, config, forge_dir)
  File "/Users/x/anaconda/lib/python2.7/site-packages/conda_smithy/configure_feedstock.py", line 77, in render_run_docker_build
    forge_config.get('channels', {}).get('sources', tuple())
  File "/Users/x/anaconda/lib/python2.7/site-packages/conda_smithy/configure_feedstock.py", line 565, in compute_build_matrix
    mtx = special_case_version_matrix(meta, index)
  File "/Users/x/anaconda/lib/python2.7/site-packages/conda_build_all/version_matrix.py", line 212, in special_case_version_matrix
    py_vn = minor_vn(index[python_pkg.fn]['version'])
AttributeError: 'Dist' object has no attribute 'fn'
```

**Comment by jakirkham** [comment]
Could you please raise an issue with this info at [conda-smithy]( https://github.com/conda-forge/conda-smithy )?

**Comment by jakirkham** [comment]
Appears re-rendering has already occurred in PR ( https://github.com/conda-forge/tzlocal-feedstock/pull/3 ), which has been merged. Closing and reopening to restart CIs w.r.t. the updated `master`.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,17 +1,17 @@
-{% set version = "1.2.2" %}
+{% set name = "tzlocal" %}
+{% set version = "1.3" %}
 {% set hash_type = "sha256" %}
-{% set hash_val = "8e7fb9f9301039c36fb0ff0a004ec36705ac46cee99a7373694a838adc8d30b0" %}
+{% set hash_val = "ed77c6c272cecca220fce830091ec6468b677b984e1b2f1f1cfc40729728b288" %}
 
 package:
-  name: tzlocal
+  name: {{ name }}
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Having the name as a jinja variable isn't really needed as that, hopefully, won't change :smile:

**Pull request merged**